package.json

```diff
@@ -1,41 +1,15 @@
 {
-  "name": "bridge-bot",
-  "version": "0.0.2",
-  "description": "Interlay bot for testing cross-chain bridges",
-  "main": "build/index.js",
-  "typings": "build/index.d.ts",
-  "repository": "https://github.com/interlay/bridge-bot",
-  "author": "Interlay",
-  "license": "Apache-2.0",
-  "scripts": {
-    "build": "tsc -p tsconfig.json",
-    "start": "node ./build/index.js",
-    "live": "ts-node src/index.ts",
-    "test": "run-s build test:*",
-    "test:integration": "mocha test/**/*.test.ts --timeout 10000000",
-    "docker-parachain-start": "ts-node scripts/setup-parachain-docker.ts",
-    "docker-parachain-stop": "docker-compose -f local-setup/docker-compose.yml down -v"
-  },
-  "engines": {
-    "node": ">=11"
-  },
-  "engineStrict": true,
-  "dependencies": {
-    "@interlay/interbtc-api": "1.8.3",
-    "@interlay/monetary-js": "0.5.3",
-    "@types/big.js": "6.1.2",
-    "@types/node": "^14.14.31",
-    "@types/underscore": "^1.11.2",
-    "@types/yargs": "^16.0.1",
-    "big.js": "6.1.1",
-    "bitcoinjs-lib": "^5.2.0",
-    "dotenv": "^10.0.0",
-    "npm-run-all": "^4.1.5",
-    "typescript": "^4.3.2",
-    "underscore": "^1.13.1",
-    "yargs": "^17.0.1",
-    "pino": "^7.5.1"
-  },
+  "private": true,
+  "workspaces": [
+    "bots/*"
+  ],
+  
+"scripts": {
+      "test": "yarn workspaces run test",
+      "build": "yarn workspaces run build",
+      "docker-parachain-start": "ts-node scripts/setup-parachain-docker.ts",
+      "docker-parachain-stop": "docker-compose -f local-setup/docker-compose.yml down -v"
```

Comment:
Not blocking:
I'm wondering if a small script file, similar to `docker-parachain-start` would be useful. e.g. It could check for the existence of `local-setup` to avoid raising errors.

Reply:
This script action is copy-pasted from the lib, it could be useful to add this, I'll open an github issue in that repo

Reply:
Added here: https://github.com/interlay/interbtc-api/issues/580